src/test/resources/authenticationTest/global_authentication_none.yml

```diff
@@ -0,0 +1,33 @@
+openapi: 3.0.1
+info:
+  description: Testing setting of global security parameter
+  title: Test
+  version: '0.0'
+paths:
+  /test:
+    get:
+      operationId: testPath
+      parameters:
+        - in: query
+          name: testDTO
+          required: true
+          schema:
+            $ref: '#/components/schemas/testDTO'
```

Comment:
**suggestion:** Use a simple type, like a string, to keep the YAML file shorter and the test case more focused.

Reply:
good point